src/passes/Print.cpp

```diff
@@ -2832,6 +2833,11 @@ struct PrintSExpression : public UnifiedExpressionVisitor<PrintSExpression> {
       *curr, [&](Memory* memory) { visitMemory(memory); });
     ModuleUtils::iterDefinedTables(*curr,
                                    [&](Table* table) { visitTable(table); });
+    for (auto name : TableUtils::getFunctionsNeedingElemDeclare(*curr)) {
+      doIndent(o, indent);
+      printMedium(o, "(elem")
+        << " declare func $" << name << ')' << maybeNewLine;
+    }
```

Comment:
Instead of printing a separate declarative segment for each function, can we put all the functions into one declarative segment?

Reply:
We can, yes. I was copying from a partner source file in which there was one each, but I'll do it that way.

First, though, what have we decided on the order? Do we need this PR?

Reply:
Fixed, now this emits a single elem in the text format.